### PR TITLE
Exposes removeFlash

### PR DESCRIPTION
**Pull request by jayseeg**
**Related**
* https://github.com/LaunchPadLab/redux-flash/issues/1

**Description**
Exposes `removeFlash` as `removeMessage` alias. `removeMessage` accepts a flash message (although it only needs the id in an object), and filters it out of the store. This exposed action allows for dismissible alerts.

### CODE REVIEW

**Comment by jayseeg** [comment]
cc: dpikt

**Comment by jayseeg** [comment]
cc: @dpikt

**Comment by dpikt** [comment]
@jayseeg this is a good enhancement, thanks! 

I've also been meaning to change `removeFlash` to receive just the message id, since it doesn't care about the rest of the object. Would you mind adding that to this PR?

**Comment by jayseeg** [comment]
@dpikt—sure thing that'll remove some extraneous objects from my app as well

**Comment by jayseeg** [comment]
@dpikt—dropped the object wrapping `id`

**Comment by jayseeg** [comment]
@dpikt—while I'm messing with `id`, what do you think of me adding a more robust unique id generator,... I'm running into identical `id`'s locally?

I can writeup another PR, or I can pile on here,... up to you?

**Comment by dpikt** [comment]
@jayseeg looks great, swapping out the id generator is fine by me. Could you add a mention of the new exposed function in the readme? After that this will be good to go.

Also don't worry about squashing, I'll do that when I merge.

**Comment by jayseeg** [comment]
@dpikt—I've updated the readme

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ getFlashMessages(state)
 - `flashErrorMessage(message, options)`: Create a flash message with the `isError` option flag set to `true`.
 - `flashSuccessMessage(message, options)`: An alias for `flashMessage()`.
 - `clearMessages()`: Clear all flash messages.
+- `removeMessage(messageID)`: Clear a particular flash messages.
```
Comment (dpikt) [inline README.md]: Should be "message" singular

Comment (jayseeg) [reply README.md]: Derp,... updated

**Comment by jayseeg** [comment]
Awesome, thanks for the quick merge @dpikt!

**Comment by dpikt** [comment]
@jayseeg no problem! Should be published soon :-)

**Pull request merged**